openpype/plugins/publish/validate_unique_subsets.py

```diff
@@ -49,17 +50,27 @@ def process(self, context):
                                  "{}".format(instance.name))
                 continue
 
-            instance_per_asset_subset[(asset, subset)].append(instance)
+            # for cases where multiple versions under same subset are published
+            version = instance.data.get("version")
+            if version is None:
+                self.log.warning("Instance found without `version` data: "
+                                 "{}".format(instance.name))
+                continue
```

Comment:
I believe this is actually quite common, right? (When follow workfile version is disabled?)

Reply:
Yes you are right, will change it via 6951af88a5fb35b70bce8769339a4dcbef7de998